llvm/test/MC/MachO/linker-options.ll

```diff
@@ -6,8 +6,6 @@
 
 ; RUN: llc -O0 -mtriple=x86_64-apple-darwin -filetype=obj -o - %s | llvm-readobj --macho-linker-options - > %t
 ; RUN: FileCheck --check-prefix=CHECK-OBJ < %t %s
-; RUN: mkdir -p %t.dir/cas
```

Comment:
Can we not remove this test coverage? But this should really go into `MC/CAS` directory in a separate test.

Reply:
Done, llvm/test/MC/MachO/linker-options.ll